extensions/vscode/package.json

```diff
@@ -538,6 +538,7 @@
   "dependencies": {
     "@hypersphere/omnibus": "0.1.6",
     "@vscode/codicons": "^0.0.36",
+    "async-mutex": "^0.5.0",
```

Comment:
New package to provide us a mutex implementation, so we could implement the throttling functionality we desired.